test/testprocess.c

```diff
@@ -485,14 +707,21 @@ static int process_testMultiprocessStdinToStdout(void *arg)
     SDL_CloseIO(input);
 
     exit_code = 0xdeadbeef;
-    buffer = (char *)SDL_ReadProcess(process2, NULL, &exit_code);
+    finished = SDL_WaitProcess(process1, true, &exit_code);
+    SDLTest_AssertCheck(finished == true, "process 1 should have finished");
+    SDLTest_AssertCheck(exit_code == 0, "Exit code of process 1 should be 0, is %d", exit_code);
+
+    exit_code = 0xdeadbeef;
+    buffer = (char *)SDL_ReadProcess(process2, &total_read, &exit_code);
```

Comment:
@sezero 
I *think* this missing wait on process 1 fixes the ci errors (I didn't stress this).
I'm not 100% sure though.

Reply:
If you rebase to current master and re-run the CI multiple times, it may give an idea.

How far is this patchset from being mergable, anyway?

Reply:
stress test: https://github.com/madebr/SDL/actions/runs/11096307767

It's conflict-free at the moment, so mergeable.
**But** there's still a blocking issue with macos https://github.com/libsdl-org/SDL/pull/10836#issuecomment-2377234165

Reply:
To unblock things, I'll modify the suite to not run the failing test, and open an issue.

Reply:
I'm afraid this is not a fix.
It still reproduces on MSVC x86:
https://github.com/madebr/SDL/actions/runs/11096307767/job/30826031525#step:25:47968